variables.tf

```diff
@@ -0,0 +1,32 @@
+variable "SECRET_KEY" {
+  description = "Required secret key"
+  type        = string
+  sensitive   = "true"
+}
+
+variable "MONGO_DATABASE_NAME" {
+  description = "Name of the database"
+  type        = string
+  sensitive   = "true"
```

Comment:
nitpick: this is an example of a non-secret value. It would be appropriate to not mark it as sensitive.